### DISCUSSION TITLE
Recommendations in Android TV Home screen

### DISCUSSION THREAD

**Discussion by samandarkuchkarov** [react-native-tvos/react-native-tvos#403] [Feature Requests]
### Description

The Android framework assists with minimum-input interaction by providing a recommendations row on the home screen. Content recommendations appear as the first row of the TV home screen after the first use of the device. Contributing recommendations from your app's content catalog can help bring users back to your app.

### Version

react-native-tvos@0.64.2-4

### Output of `npx react-native info`

System:
    OS: Windows 10 10.0.19043
    CPU: (16) x64 11th Gen Intel(R) Core(TM) i7-11700 @ 2.50GHz
    Memory: 6.38 GB / 15.80 GB
  Binaries:
    Node: 16.14.2 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.5 - C:\Program Files (x86)\Yarn\bin\yarn.CMD    
    npm: 8.5.0 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  SDKs:
    Android SDK:
      API Levels: 29, 30, 31
      Build Tools: 29.0.2, 30.0.2
      System Images: android-29 | Android TV Intel x86 Atom, android-29 | Intel x86 Atom_64, android-29 | Google APIs Intel x86 Atom, android-29 | Google Play Intel x86 Atom
      Android NDK: 22.1.7171670
    Windows SDK: Not Found
  IDEs:
    Android Studio: Version     2020.3.0.0 AI-203.7717.56.2031.7621141
    Visual Studio: Not Found
  Languages:
    Java: 11.0.13 - C:\Program Files\Common Files\Oracle\Java\javapath\javac.EXE
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.1 => 17.0.1
    react-native: Not Found
    react-native-tvos:  0.64.2-4
    react-native-windows: Not Found
  npmGlobalPackages:
    *react-native*: Not Found

### Steps to reproduce

Is there a way of creating recommendation row videos in android TV home screen

### Snack, code example, screenshot, or link to a repository

https://developer.android.com/training/tv/discovery/recommendations-row

**Comment by freeboub** [comment]
I was able to implement this feature on my project, but unfortunately, I cannot share the code.
There is no generic library to acces to android content provider in react native (or I didn't find it).
So I think you need to implement it from scratch.

**Reply by coucoseth** [reply]
@freeboub it would be great if you could take us through the implementation. i tried following the documentation but its not easy to understand. They instruct you to add a code snippet but dont tell you where to add it.
